transports/quic/src/connection.rs

```diff
@@ -66,6 +66,9 @@ pub enum Error {
     /// Endpoint has force-killed this connection because it was too busy.
     #[error("Endpoint has force-killed our connection")]
     ClosedChannel,
+    /// The background task driving the endpoint has crashed.
+    #[error("Background task crashed.")]
```

Comment:
Nit: These are often chained and a dot in the middle may look weird.

```suggestion
    #[error("Background task crashed")]
```